README.md

```diff
@@ -20,6 +20,9 @@ flowplayer('#player', {
     title: 'Bauhaus',
     thumbnails: {
       template: 'thumbnails/bauhaus{time}.jpg'
+      time_format: function(t) {
+        return t + "-thumb.jpg";
```

Comment:
Could you give an example or use case where the function is _needed_?
The above could be simply set with `template: 'thumbnails/bauhaus{time}-thumb.jpg'`, couldn't it?